### ISSUE TITLE
Problem with SSPXr

### ISSUE THREAD

**Issue by NomenNescio92** [ScienceFun/Kerbalism-Continued#14]
Stockalike Station Parts Expansion Redux

CTD-5 ' Doughnut' Compact Inflatable Centrifuge Module
CTD-10 'Bagel' Inflatable Centrifuge Module
PFD-C 'Coriolis' Inflatable Centrifuge Module
PXL-F 'Pilgrim' Extensible Centrifuge
PXL-E 'Mercury' Extensible Centrifuge

The centrifuges don't appear to add any "living space" to the craft. The PXL-E 'Mercury' Extensible Centrifuge has exercise equipment in it, but doesn't count towards "Exercise" in the kerbalism menu.

**Comment by NomenNescio92** [comment]
Also the centrifuges have an option to "inflate" and to "start rotation". But it's not until you hit the button marked "deploy" that the bonuses take effect. Even if the thing is spinning, it doesn't take up power or give the "firm ground" bonus.

**Comment by HaullyGames** [comment]
Need implement Inflatable for:
CTD-10 Inflatable Centrifuge Module = sspx-inflatable-centrifuge-125-1
CTD-5 Compact Inflatable Centrifuge Module = sspx-inflatable-centrifuge-125-2
PFD-C Inflatable Centrifuge Module = sspx-inflatable-centrifuge-25-1
PXL-F 'Pilgrim' Extensible Centrifuge = sspx-expandable-centrifuge-375-2
PXL-E 'Mercury' Extensible Centrifuge = sspx-expandable-centrifuge-375-1

**Comment by HaullyGames** [comment]
Duplicate of https://github.com/ScienceFun/Kerbalism-Continued/issues/10
